cmd/context/context.go

```diff
@@ -58,6 +59,8 @@ This will prompt you to select one of your existing contexts or to create a new
 	cmd.Flags().StringVarP(&ctxOptions.Namespace, "namespace", "n", "", "namespace of your okteto context")
 	cmd.Flags().StringVarP(&ctxOptions.Builder, "builder", "b", "", "url of the builder service")
 	cmd.Flags().BoolVarP(&ctxOptions.OnlyOkteto, "okteto", "", false, "only shows okteto cluster options")
-	cmd.Flags().MarkHidden("okteto")
+	if err := cmd.Flags().MarkHidden("okteto"); err != nil {
+		oktetoLog.Infof("failed to mark 'okteto' flag as hidden: %s", err)
```

Comment:
@jLopezbarb should this be a `Debug` log?

Reply:
I'd leave it as Info, debug is not stored in log file which makes it harder to detect